### PR TITLE
refactor: part of general improvement

### PR DESCRIPTION
**Pull request by momshaddinury**
### Issue?
- Part of #43 
### What was done and why?
- unused authentication related old files removed
- calendar screen and related widgets moved to feature directory

### CODE REVIEW

File: lib/UI/prev_workout/prev_workout_page.dart [hunk 1]
```diff
@@ -83,7 +83,7 @@ class _PrevWorkoutPageState extends State<PrevWorkoutPage> {
                     Navigator.of(context).pop();
                     Navigator.of(context).push(
                       MaterialPageRoute(
-                        builder: (context) => CalendarPage(),
+                        builder: (context) => CalendarScreen(),
```
Comment (jorre127) [inline lib/UI/prev_workout/prev_workout_page.dart:86]: :pray:

**Review by jorre127 — APPROVED** [review]

File: lib/UI/prev_workout/prev_workout_page.dart (hunk 1 above)
Comment (momshaddinury) [reply lib/UI/prev_workout/prev_workout_page.dart:86]: We need to work on navigation. Do you have any idea about how many screens we have?

Comment (jorre127) [reply lib/UI/prev_workout/prev_workout_page.dart:86]: Not a lot I think only 3? authentication screen, calendar screen and workout screen.

Comment (jorre127) [reply lib/UI/prev_workout/prev_workout_page.dart:86]: how are we gonna handle navigation? Define named routes in the MaterialApp? or make our own navigator.

**Pull request merged**